libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -201,24 +206,20 @@ export class DtStackedSeriesChart implements OnDestroy {
 
   /** Current selection [series, node] */
   @Input()
-  get selected(): [DtStackedSeriesChartSeries, DtStackedSeriesChartNode] | [] {
+  get selected(): DtStackedSeriesChartSelection | [] {
```

Comment:
Changing the type would be a breaking change. Please do it without changing the public API surface.